src/main/java/com/skynest/uitesting/tests/UserRegistrationTest.java

```diff
@@ -0,0 +1,28 @@
+package com.skynest.uitesting.tests;
+
+import com.skynest.uitesting.models.User;
+import com.skynest.uitesting.pages.RegistrationForm;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static com.skynest.uitesting.constants.PageUrlConstants.LOGIN_URL;
+
+public class UserRegistrationTest extends BaseTest {
+
+    @Test
+    public void registering_new_valid_user_should_navigate_to_login_page() {
+        registrationPage.openPage();
+        User user = User.generateValidAccount();
+        RegistrationForm registrationForm = registrationPage.fillRegistrationForm()
+                .withFirstName(user.getFirstName())
+                .withLastName(user.getLastName())
+                .withEmail(user.getEmailAddress())
+                .withPhoneNumber(user.getPhoneNumber())
+                .withHomeAddress(user.getHomeAddress())
+                .withPassword(user.getPassword())
+                .withConfirmPassword(user.getConfirmPassword());
+        registrationPage.scroll(0, 400);
+        registrationForm.submitForm();
+        waitForUrl(LOGIN_URL);
```

Comment:
It could be the case that your browser navigates to the correct url, but ,for whatever reason, the page is not rendered correctly. It would be preferable to assert that the login form is actually present (and can be interacted with).